apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/viewmodel/HomeEventNameViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.twilio.livevideo.app.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class HomeEventNameViewModel @Inject constructor() : ViewModel() {
+
+    val newEventName = MutableLiveData<String>()
+
+    val joinEventName = MutableLiveData<String>()
```

Comment:
Do you think these can just be combined to be `eventName`? Or is there a reason that doesn't make sense?

Reply:
I though of that, but... in this case, these attributes are 2-way-binded with the UI, which means the `EditTextView` in the layout is attached to this attribute in order to capture the data in the `ViewModel`. 
So, if I use one single attribute, the same attribute would be shared with two screens (`Create new event` and `Join event`) and the data is persisted in the ViewModel... 

So, let's say if the user enter an event name in `Create new event` screen but decide to go back and go to `Join event` screen, then the typed name in the previous screen would be there as well. :)

Reply:
Ok makes sense thanks!